ops/solana/solana.go

```diff
@@ -265,23 +267,43 @@ func (d *Deployer) DeployOCR() error {
 	err = d.gauntlet.ExecCommand(
 		"store:set_writer",
 		d.gauntlet.Flag("network", d.network),
-		d.gauntlet.Flag("ocrState", d.Account[OCRFeed]),
 		d.gauntlet.Flag("input", string(jsonInput)),
+		d.Account[OCRFeed],
 	)
 
 	if err != nil {
 		return errors.Wrap(err, "setting writer on store failed")
 	}
+
+	fmt.Println("Step 8: Transfer feed ownership to store")
+	if err = d.gauntlet.ExecCommand(
+		"store:transfer_feed_ownership",
+		d.gauntlet.Flag("network", d.network),
+		d.gauntlet.Flag("state", d.Account[OCRTransmissions]),
+		d.gauntlet.Flag("to", d.Account[StoreAccount]),
+	); err != nil {
+		return errors.Wrap(err, "failed to transfer feed ownership")
```

Comment:
```suggestion
		return errors.Wrap(err, "'store:transfer_feed_ownership' call failed")
```

If this is surfaced through an original error, maybe even is unnecessary to wrap.

Reply:
true! personally, I like wrapping it since it's more explicit and makes things easier to trace since this is primarily a testing env 🤷🏼